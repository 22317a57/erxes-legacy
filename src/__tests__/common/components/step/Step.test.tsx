import { mount, shallow } from 'enzyme';
import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Step from '../../../../modules/common/components/step/Step';

describe('Step component', () => {
  const defaultProps = {
    children: false
  };

  test('renders Step successfully', () => {
    shallow(<Step {...defaultProps} />);
  });

  test('renders successfully with default value', () => {
    const wrapper = mount(<Step {...defaultProps} />);
    const props = wrapper.props();

    expect(props).toMatchObject(defaultProps);
  });

  test('snapshot matches', () => {
    const rendered = renderer.create(<Step {...defaultProps} />).toJSON();

    expect(rendered).toMatchSnapshot();
  });
});
