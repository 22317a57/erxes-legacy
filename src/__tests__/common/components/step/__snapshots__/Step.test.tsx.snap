// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Step component snapshot matches 1`] = `
<div
  className="sc-elNKlv cXNnps"
>
  <div
    className="sc-hUMlYv gKRuHB"
  >
    <div
      className="sc-ESoVU bGlnRn"
    >
      <div
        className="sc-kkbgRg bObUvT"
      >
        <div
          className="sc-hRmvpr gdtmAR"
        >
          <img
            alt="step-icon"
          />
        </div>
        <h5
          className="sc-cZBZkQ bpuhcF"
        >
          
        </h5>
      </div>
    </div>
    <div
      className="sc-ecaExY frzVdF"
    />
  </div>
  <div
    className="sc-gbzWSY jqsjmA"
    onClick={[Function]}
  >
    <div
      className="sc-hRmvpr gdtmAR"
    >
      <img
        alt="step-icon"
      />
    </div>
  </div>
</div>
`;
